### DISCUSSION TITLE
Presets + Bookmarks Table showing nulls for global scoped presets

### DISCUSSION THREAD

**Discussion by colemahatmccreary** [directus/directus#14015] [Feature Requests]
Not displaying any fields other than the ID. See below:

<img width="941" alt="Screen Shot 2022-06-20 at 11 13 27 AM" src="https://user-images.githubusercontent.com/40645344/174658406-b0f039b3-5f22-45c7-b1a7-448ed2be139c.png">

**Comment by azrikahar** [comment]
#10978 change Presets & Bookmarks page to use layout wrapper which in turn only reflects the actual columns within the table (vs how it used to use "fake" table/columns to represent scopes), and they are indeed `null` when there are no bookmark name, no role, and no user. This in turn implicitly means it's a `Global` scoped preset.

Here's an example on the differences:

https://user-images.githubusercontent.com/42867097/174733309-d026705a-a467-4c7b-be99-eef74d6e89e2.mp4

Understandably an alternative way to reflect it as a global scoped preset would be better, hence moved this to a discussion for now as it's not an inherent bug.

**Reply by colemahatmccreary** [reply]
Not exactly clear what the video is supposed to show – how do i manage global/default presets and bookmarks?

**Reply by azrikahar** [reply]
The ones with null bookmark + global scope should be global presets, whereas _named_ bookmark + global scope will be treated as global bookmarks. Are you having any issues configuring them? Or do you mean something else by "manage"?

In the video below, we can see when I added a new column to the **global preset**, the new column is shown. And when I give it a name (icon/color configuration is optional), it now appears as a **global bookmark**. Hope that clears things up.

https://user-images.githubusercontent.com/42867097/177065276-c1cd5c3c-a08e-435b-9a9a-c767fdf5e2fd.mp4

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
